src/main/java/space/obminyashka/items_exchange/service/impl/AuthServiceImpl.java

```diff
@@ -30,25 +26,18 @@ public class AuthServiceImpl implements AuthService {
 
     private final RefreshTokenService refreshTokenService;
     private final JwtTokenService jwtTokenService;
-    private final UserService userService;
-    private final UserMapper userMapper;
 
-    public UserLoginResponseDto createUserLoginResponseDto(String username) throws UsernameNotFoundException {
-        return userService.findByUsernameOrEmail(username)
-                .map(this::populateUserLoginResponseDto)
-                .orElseThrow(() -> new UsernameNotFoundException("User" + username + " is not logged in"));
-    }
+    public UserLoginResponseDto finalizeAuthData(UserLoginResponseDto userDto) {
+        userDto.setAccessToken(jwtTokenService.createAccessToken(userDto.getUsername(), userDto.getRole()));
+        userDto.setAccessTokenExpirationDate(jwtTokenService.getAccessTokenExpiration(userDto.getAccessToken()));
+        var refreshToken = refreshTokenService.createRefreshToken(userDto.getRefreshToken(), userDto.getUsername());
+        userDto.setRefreshToken(refreshToken.getToken());
```

Comment:
Previously setting of variables into the DTO was created with population as chains (see previous commits).
If you wouldn't use it - remove all Accessors from the DTO or rewrite your code using previous versions of the DTO initialization.